Patch for a Home Assistant Voice PE firmware, upstream esphome/home-assistant-voice-pe
home-assistant-voice.yaml at d4e6fa43d6a1a7d342b7d4d403567e06516719de. See README.md here.
--- a/home-assistant-voice.yaml
+++ b/pi-voice-pe.yaml
@@ -35,9 +35,9 @@
   error_cloud_expired_sound_file: https://github.com/esphome/home-assistant-voice-pe/raw/dev/sounds/error_cloud_expired.mp3
 
 esphome:
-  name: home-assistant-voice
-  friendly_name: Home Assistant Voice
-  name_add_mac_suffix: true
+  name: pi-voice-pe
+  friendly_name: Pi Voice PE
+  name_add_mac_suffix: false
   min_version: 2026.9.0
   on_boot:
     priority: 375
@@ -82,6 +82,8 @@
 
 wifi:
   id: wifi_id
+  ssid: !secret wifi_ssid
+  password: !secret wifi_password
   fast_connect: ${hidden_ssid}
   on_connect:
     - lambda: id(improv_ble_in_progress) = false;
@@ -111,7 +113,8 @@
     - script.execute: control_leds
   on_client_disconnected:
     - script.execute: control_leds
-  encryption:   # Uses key set by Home Assistant
+  encryption:
+    key: !secret api_key
 
 ota:
   - platform: esphome
